packages/headless/src/app/navigatorContextProvider.ts

```diff
@@ -20,6 +20,15 @@ export interface NavigatorContext {
    * See [clientId](https://docs.coveo.com/en/masb0234).
    */
   clientId: string;
+  /**
+   * Whether to capture analytics events.
+   *
+   * Must be set left undefined or set to `false` if the `clientId` is an empty string, otherwise requests will fail.
+   *
+   * Should also be set to `false` to comply with regulations if the user has somehow indicated that they do not wish to
```

Comment:
Regulations varies depending on regions and all, let's not provide legal advices and focus on what the parameter does only.

Reply:
@jpmarceau what do you think? Is there some documentation we could/should link to instead?

Reply:
You could link to https://docs.coveo.com/en/3348
But I think it would be fair to do as Louis suggests, only stating what the param does. I would expect readers to consult the documentation site rather than reference documentation for such considerations.